octopoes/octopoes/core/service.py

```diff
@@ -199,12 +199,10 @@ def _run_inference(self, origin: Origin, valid_time: datetime) -> None:
                 config = configs[-1].config
 
         try:
-            resulting_oois = BitRunner(bit_definition).run(source, list(parameters.values()), config=config)
+            resulting_oois = BitRunner(bit_definition).run(source, parameters, config=config)
+            self.save_origin(origin, resulting_oois, valid_time)
```

Comment:
I don't understand why save_origin is moved inside the try/except, if save_origin throws an exception I don't think we should silence that.